lib/api/core/httpRouter/index.js

```diff
@@ -136,6 +138,8 @@ class Router {
       return replyWithError(cb, request, new BadRequestError(`Unrecognized HTTP method ${httpRequest.method}`));
     }
 
+    httpRequest.url = httpRequest.url.replace(LeadingSlashRegex, '');
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ASemicolon)